src/index.js

```diff
@@ -73,7 +73,10 @@ export type {IconAsButtonPropsType} from './components/icon-as-button/IconAsButt
 export {default as Icon} from './components/icons/Icon';
 export type {IconPropsType} from './components/icons/Icon';
 export {default as MobileIcon} from './components/mobile-icons/MobileIcon';
-export type {MobileIconPropsType} from './components/mobile-icons/MobileIcon';
+export type {
+  MobileIconPropsType,
+  IconTypeType as MobileIconTypeType,
```

Comment:
How about changing naming to something like: `MobileIconTypeType`?

Reply:
No problem for me, but keep in mind that all types should have Type postfix and icon type apparently ends the same 😄 It will pass linting but can be less consistent with other type names

Reply:
True, but I would rather remove Type suffix everywhere. It is there only because of flow. Currently we export only 2 kind of types. PropsType for each component and now second one for Icons names. All the other types are access by index and we dont really care about their names.

Reply:
Yes, I feel the same :) I changed both exported types to `MobileIconType` and `SubjectIconType`